### PR TITLE
Code cleanup, debug traces added and reenabling FAT

### LINKED ISSUE #29169: OpenTelemetry Logging Code Cleanup
A few things need to be cleaned up and renamed but we wanted to guarantee the Logging code would make the upcoming beta release. 

Use Tr isntead of printStackTrace for the exception logging, rename variables, and minor cleanup.

See #29108 for specific changes requested.

### PR DESCRIPTION
**Pull request by tonyreigns**
Fixes #29169

### CODE REVIEW

**Comment by tonyreigns** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck,io.openliberty.http.monitor_fat,io.openliberty.microprofile.telemetry.2.0.logging.internal_fat

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/MpTelemetryLogMappingUtils.java [hunk 1]
```diff
@@ -259,6 +259,7 @@ private static String formatExtensionKey(String extKey) {
         return sb.toString();
     }
 
+    //Removes leading new line spaces from strings
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/MpTelemetryLogMappingUtils.java]: Can you make this comment wrapped around /**/ instead, to make it consistent.

**Review by pgunapal — CHANGES_REQUESTED** [review]
Minor comment update

**Review by pgunapal — DISMISSED** [review]
LGTM

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
None of the test failures are related to these changes. Message verification was resolved after this build started.

**Review by pgunapal — APPROVED** [review]
Looks good!

**Pull request merged**